lib/KBaseReport/utils/report_utils.py

```diff
@@ -18,7 +18,8 @@ def create_report(params, dfu):
     workspace_id = _get_workspace_id(dfu, params)
     # Empty defaults for merging
     report_data = {
-        'objects_created': []
+        'objects_created': [],
+        'text_message': '',
```

Comment:
ensure that the workspace doesn't complain if we upload a report with `direct_html` and no `text_message`